vllm/model_executor/custom_op.py

```diff
@@ -138,15 +156,22 @@ def default_on() -> bool:
     # - MyOp.enabled()
     # - op_registry["my_op"].enabled()
     op_registry: dict[str, type['CustomOp']] = {}
+    op_registry_oot: dict[str, type['CustomOp']] = {}
 
     # Decorator to register custom ops.
     @classmethod
-    def register(cls, name: str):
+    def register(cls, name: str, custom_op=False):
 
         def decorator(op_cls):
-            assert name not in cls.op_registry, f"Duplicate op name: {name}"
-            op_cls.name = name
-            cls.op_registry[name] = op_cls
+            if custom_op:
+                assert name not in cls.op_registry_oot, \
```

Comment:
I think it's better to name this arg to `is_oot_custom_op`
```suggestion
    def register(cls, name: str, is_oot_custom_op=False):
```

Reply:
Thanks, Arg name updated.